src/wallet/gtest/test_wallet.cpp

```diff
@@ -1202,7 +1219,7 @@ TEST(WalletTests, CachedWitnessesChainTip) {
 
     {
         // Second transaction
-        auto wtx = GetValidSproutReceive(sk, 50, true, 4);
+        auto wtx = GetValidSproutReceive(sk, 50, true);
```

Comment:
I see this fix to the call site here (and elsewhere) but was changing the definition ever rejected as a safer way forward?